Joinpoint Regression in R.R

```diff
@@ -61,133 +73,158 @@ ggplot(mortdata, aes(year, asdr, color = cod)) +
   theme_classic() +
   xlab("Year") +
   ylab("Age-Adjusted Death Rate (per 100,000)") +
-  tayloRswift::scale_color_taylor("lover") # yes, this IS a Github repo dedicated to color palettes of Taylor Swift albums
+  scale_color_taylor("lover")
```

Comment:
Package name no longer needs to be referenced as it is now in the global environment namespace